### DISCUSSION TITLE
[Suggestion] Adding "Not Interested" / "Don't Recommend this User" button

### DISCUSSION THREAD

**Discussion by lilisako** [community/community#13243] [Other Feature Feedback, Questions, & Ideas]
Hi, community!!👋

This "For you" feed could be super useful for searching for something new, but it would be nice if there is "Not Interested" or "Don't Recommend this User" button. 

Thanks!

**Comment by sntwandy** [comment]
As to have a ranking of the most recommended content for all of us, so filter the less interesting content.

**Comment by bdresser** [comment]
Thanks @lilisako! This is great feedback. We do have plans to let folks provide feedback on the individual content and recommendations they're seeing so that you can help tailor the feed to what you're interested in. Stay tuned :)

**Reply by lilisako** [reply]
Thanks for answering! I'm looking forward to the update :))

**Reply by Gkolachi** [reply]
Thank you

**Reply by Cherie1984** [reply]
Thank you

**Comment by mdsumner** [comment]
yes please, every day there's a user I do not want to see being "Recommended based on people you follow" - and I'd be very happy to click "do not show me this user"

**Reply by Gkolachi** [reply]
Thank you

**Comment by MichaelWheldon** [comment]
Think about what you’re asking... you’re wanting a feature which makes _you_ go out of _your_ way to remove something you didn’t add? Like, this is why LinkedIn etc are now completely unusable. It’s just a never ending game of plugging up holes and hiding people and content you don’t want to see.

This is a negative interaction you’re being funnelled into, and for what?

**Reply by l-zeuch** [reply]
Respectfully, I think you're missing the point.

There are situations where it is necessary for you to interact with a user, for instance in case of spam, being overly aggressive, not following both GitHub and project community guidelines.

This causes the "For You" feed to consider it as a collaboration, when in reality, it is not a collaboration. With this in mind, the suggestion is valid and has a real use-case: Not seeing people who are hostile and not worth anyone's time.

I don't want to see the activity of users I do not wish nor ever plan to interact with.

**Reply by MichaelWheldon** [reply]
I get that, but was making the point that people suggesting this are asking for a negative UI interaction to solve something they didn’t opt into. This tells me this whole feature is ill conceived.

I understand why something like this might be needed, but its need is forced upon us through this stupid feed.

**Reply by l-zeuch** [reply]
I mean, in the end, you can choose to *not* open the feed :shrug: 

Personally I don't really use it and only take a look when I'm bored or just want to browse for a moment, but in these cases I specifically go out of my way hunting for content I might like. 
Clicking "not interested" will improve my feed over time, I could imagine.

**Reply by MichaelWheldon** [reply]
"Clicking "not interested" will improve my feed over time, I could imagine." You’re wasting your life doing that interaction lol Admittedly it takes a couple seconds but it’s an empty interaction _at best_ which is kind of my point. These algorithmic feeds are designed to waste your life.

And I have noticed things spilling into the main feed now. If someone I follow follows someone else I get that pushed into the "Following" feed, so no, I can not ignore it.

**Reply by l-zeuch** [reply]
> You're wasting your life doing that interaction

As I already established, I specifically go out of my way browsing the feeds, so this point is moot in my case :^)
Also I fail to see how it would be an "empty" interaction? Given it is properly implemented (unlike on YouTube with their disfunctional "Don't recommend this channel"), it certainly won't be empty, as it has an effect.

> And I have noticed things spilling into the main feed now. If someone I follow follows someone else I get that pushed into the "Following" feed, so no, I can not ignore it.

I've not noticed anything like that so far, but it could very well be. If that is case, that's a shame and GitHub should ensure that only contents you're directly following shows up in the "Following" tab.

**Reply by Gkolachi** [reply]
Really good feed back.self inventory is always helpful. Thank you

**Comment by zkamvar** [comment]
In my specific use-case, I often see the profile of someone who was abusive to me, but because I have worked closely with them in the same field, their profile and activity get recommended to me _a lot_ in this feed. Having a mechanism to prevent this from happening in the first place (e.g. a feed blocklist in the settings) would really help my sanity.

**Reply by siminapasat** [reply]
@zkamvar Thanks for letting us know! Have you blocked this user? (You can do so by viewing the profile and then hitting "Block or report" from the left-hand-side column.

**Reply by zkamvar** [reply]
This is a situation where I still need to interact with this user in some instances and outright blocking would prevent these interactions. A block or no block option for moderating interaction with some folks is kinda like living in an apartment and having the choice of removing the door or nailing it shut.

**Reply by foursixnine** [reply]
I'm here with @zkamvar, I'm seeing people I don't want to see, but don't necessarily want/can block them (Which is even a bigger issue when you're  working on multiple Open Source/Upstream projects, hosted on gh...

**Comment by emilqnn** [comment]
по добре да има нещо като черен списък личен и там да си добавиш кого не искаш да виждаш  и да са синхронизира с профила ти

**Reply by Gkolachi** [reply]
Does the platform have translation? Just asking?😊 Thank you

**Comment by yduman** [comment]
It would be also nice if I can e.g. "hide all release notifications". I'm mostly starring repos that I come back to later or that I find interesting. Now in the "For you" section, I get notifications about each release of all repos I starred. To be honest, I don't care if these projects published a new release. If I'm curious about a new release of a project, I simply visit the repo page itself or hit the "watch" button on repos I want to follow. Hope that makes sense :)

**Reply by siminapasat** [reply]
We have this planned to go out tomorrow 🤞

**Reply by yduman** [reply]
just saw it! thank you so much!! it's much better now :)

**Comment by jfullstackdev** [comment]
hello guys, there is a new update on the For You feed, you can now filter it, and for those who don't want it can simply disable it by unchecking all filters

![image](https://user-images.githubusercontent.com/47092464/182012117-0b19be9d-e849-462e-909e-d20d28915c1d.png)
